fastcrypto/src/secp256k1/recoverable.rs

```diff
@@ -33,6 +33,7 @@ use rust_secp256k1::{
     All, Message, Secp256k1,
 };
 use std::fmt::{self, Debug, Display};
+pub use rust_secp256k1::ecdsa::Signature as secpSig;
```

Comment:
`as secp256k1Sig` not to be confused with `secp256r1` that we also support

Reply:
makes sense, updated it now